embedded_sass.proto

```diff
@@ -224,12 +224,11 @@ message InboundMessage {
 
   // The wrapped message. Mandatory.
   oneof message {
-    ProtocolError error = 1;
-    CompileRequest compileRequest = 2;
-    CanonicalizeResponse canonicalizeResponse = 3;
-    ImportResponse importResponse = 4;
-    FileImportResponse fileImportResponse = 5;
-    FunctionCallResponse functionCallResponse = 6;
+    CompileRequest compileRequest = 1;
+    CanonicalizeResponse canonicalizeResponse = 2;
+    ImportResponse importResponse = 3;
+    FileImportResponse fileImportResponse = 4;
+    FunctionCallResponse functionCallResponse = 5;
```

Comment:
Generally when a protocol buffer field is removed, the other field indexes are left as-is so that older clients will continue to work. We don't really have any old clients in the wild, but since we do have releases up on GitHub it's probably still a good idea to leave the indexes.

Reply:
Good to know.